### PR TITLE
Chore/update integration tests expired token expected response

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem

API endpoints for uploading `documents`/`live_photos`/`live_videos` has changed to return no info in the error response 
on attempting to upload a file with an expired JWT token. This causes the integration tests for expired token scenario to fail and hence CI build also fails.

# Temporary Solution

Update expected error response for document, selfie, video integration tests to unblock builds while API is fixed to return the intended (new) error response.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by ethan605 — CHANGES_REQUESTED** [review]
Humm tbh the changes look just okay, but I don't like it, so many changes involved having a _temporary unblock_ of CI.

IMO we can do a bit differently:
- Instead of export `TEMP_EMPTY_ERROR`, let's export `EXPECTED_EXPIRED_TOKEN_ERROR` again
- Within `mockExpiredJwtAndResponse.js`, prepend the real `EXPECTED_EXPIRED_TOKEN_ERROR` with a `_` or `ORIGINAL_`
- Change `EXPECTED_EXPIRED_TOKEN_ERROR` to the empty values

**Comment by kopijunkie** [comment]
> Humm tbh the changes look just okay, but I don't like it, so many changes involved having a _temporary unblock_ of CI.
> 
> IMO we can do a bit differently:
> 
> * Instead of export `TEMP_EMPTY_ERROR`, let's export `EXPECTED_EXPIRED_TOKEN_ERROR` again
> * Within `mockExpiredJwtAndResponse.js`, prepend the real `EXPECTED_EXPIRED_TOKEN_ERROR` with a `_` or `ORIGINAL_`
> * Change `EXPECTED_EXPIRED_TOKEN_ERROR` to the empty values and keep the inline documentation for the future fixes

Good suggestion! I wasn't very comfortable with the amount of changes for this too but couldn't think of a better way.

File: src/components/utils/__integrations__/onfidoApi/selfie.integration.js [hunk 1]
```diff
@@ -155,7 +156,7 @@ describe('API uploadSnapshot endpoint', () => {
     expect.hasAssertions()
     const onErrorCallback = (error) => {
       try {
-        expect(error).toEqual(EXPECTED_EXPIRED_TOKEN_ERROR)
+        expect(error).toEqual(ORIGINAL_EXPECTED_EXPIRED_TOKEN_ERROR)
```
Comment (ethan605) [inline src/components/utils/__integrations__/onfidoApi/selfie.integration.js:159]: Should this be `EXPECTED_EXPIRED_TOKEN_ERROR`?

Comment (kopijunkie) [reply src/components/utils/__integrations__/onfidoApi/selfie.integration.js:159]: Oops. You're right!

Comment (kopijunkie) [reply src/components/utils/__integrations__/onfidoApi/selfie.integration.js:159]: Not for `/snapshot` endpoint. That is still returning the original `expired_token` error response.

Comment (ethan605) [reply src/components/utils/__integrations__/onfidoApi/selfie.integration.js:159]: Damn this is annoying 😅

**Review by ethan605 — APPROVED** [review]
LGTM!

**Review by stefaniacardenas — APPROVED** [review]
Is the Onfido API response breaking the SDK behaviour? This sounds like something that might be breaking several SDK versions, not only for the integrator but also the user.

File: src/components/utils/__integrations__/onfidoApi/selfie.integration.js (hunk 1 above)
Comment (kopijunkie) [reply src/components/utils/__integrations__/onfidoApi/selfie.integration.js:159]: Agreed. 😅

**Comment by kopijunkie** [comment]
> Is the Onfido API response breaking the SDK behaviour? This sounds like something that might be breaking several SDK versions, not only for the integrator but also the user.

User will still see the generic error message on attempting upload with an expired JWT token (tried on Surge latest), so it's not breaking the SDK behaviour for end user I think. But you are right about it affected other SDK versions. Which is why I can't confirm the SDK behaviour hasn't been broken for sure.

**Pull request merged**